ArduPlane/Attitude.cpp

```diff
@@ -18,6 +18,15 @@ float Plane::get_speed_scaler(void)
             speed_scaler = 2.0;
         }
         speed_scaler = constrain_float(speed_scaler, 0.5f, 2.0f);
+
+        if (quadplane.in_vtol_mode() && hal.util->get_soft_armed()) {
+            // when in VTOL modes limit surface movement at low speed to prevent instability
+            float threshold = aparm.airspeed_min * 0.5;
+            if (aspeed < threshold) {
+                float new_scaler = linear_interpolate(0, g.scaling_speed / threshold, aspeed, 0, threshold);
```

Comment:
This propagates a value at or near 0 into the control surface PID controller. Doesn't this effectively serve to to hold whatever integrator output was last had since the error is near 0? I'm not certain if this is actually a bad thing. So far the only scenario I've come up with is that you tilted the plane while on the ground/try to take off from a not level surface, wound up an integrator while disarmed and then takeoff in quadplane auto. You could be carrying some integrator you don't want when you first transition, but it should disappear once you are in flight, assuming you don't have a very low I gain. (This does seem a bit stretched, and I thought we had some anti windup stuff for before takeoff, but I didn't find it quickly)

Reply:
we can't really fix that till we move the scaling to the output side of the PIDs